### PR TITLE
Modified sign in page 

### LINKED ISSUE #47: Application UI
As a pet owner I want the application to be simple to use so that I can activate my pet easily.

Acceptance criteria:

Final UI developed in Figma and ready for testing and implementation.

### LINKED ISSUE #58: improve UI design on sign in page
Work on design of sign in page.

### PR DESCRIPTION
**Pull request by InaJ123**
- Updated design of sign in page & removed sign out button
- Temporarily removed sign in forms that are currently not working & created an issue to fix them on git 

Resolves #58

### CODE REVIEW

File: smart_pet_buddy/lib/signin_page.dart [hunk 1]
```diff
@@ -131,8 +104,9 @@ class _EmailPasswordFormState extends State<_EmailPasswordForm> {
                 Container(
                   padding: const EdgeInsets.only(top: 16),
                   alignment: Alignment.center,
-                  child: SignInButton(
-                    Buttons.Email,
+                  child: SignInButtonBuilder(
```
Comment (emilieanthony) [inline smart_pet_buddy/lib/signin_page.dart:107]: Nice, did the SignInButton cause any restrictions in the design?

Comment (InaJ123) [reply smart_pet_buddy/lib/signin_page.dart:107]: Yes exactly! Had to change in order to be able to choose a different color of the button for example.

Comment (sofie-a) [reply smart_pet_buddy/lib/signin_page.dart:107]: Good job!

**Comment by AstridBer** [comment]
Nice work!

**Review by emilieanthony — APPROVED** [review]
Nice enhancement of the UI overall!

**Pull request merged**